classes/class-wc-connect-shipping-label.php

```diff
@@ -0,0 +1,316 @@
+<?php
+
+if ( ! class_exists( 'WC_Connect_Shipping_Label' ) ) {
+
+	class WC_Connect_Shipping_Label {
+
+		private $settings_store;
+
+		public function __construct( WC_Connect_Service_Settings_Store $settings_store ) {
+			$this->settings_store = $settings_store;
+		}
+
+		protected function get_form_data( WC_Order $order ) {
+			$formData = array();
+			$contents = array();
+
+			foreach( $order->get_items() as $item ) {
+				$product = $order->get_product_from_item( $item );
```

Comment:
I had this line fail locally when testing. Please handle the non-`WC_Product` return cases.

See: https://docs.woothemes.com/wc-apidocs/source-class-WC_Abstract_Order.html#1879-1896